config.md

```diff
@@ -7,304 +7,170 @@ This specification outlines the JSON format describing images for use with a con
-	The array is ordered from bottom-most layer to top-most layer.
-	The object has the following fields.
-
-        <ul>
-          <li>
-            <code>created</code>: Creation time, expressed as a ISO-8601 formatted
-            combined date and time
-          </li>
-          <li>
-            <code>author</code>: The author of the build point
-          </li>
-          <li>
-            <code>created_by</code>: The command which created the layer
-          </li>
-          <li>
-            <code>comment</code>: A custom message set when creating the layer
-          </li>
-          <li>
-            <code>empty_layer</code>: This field is used to mark if the history item created a filesystem diff.
-	    It is set to true if this history item doesn't correspond to an actual layer in the rootfs section (for example, a command like ENV which results in no change to the filesystem).
-          </li>
-        </ul>
-
-Here is an example history section:
-
-<pre>"history": [
-  {
-    "created": "2015-10-31T22:22:54.690851953Z",
-    "created_by": "/bin/sh -c #(nop) ADD file:a3bc1e842b69636f9df5256c49c5374fb4eef1e281fe3f282c65fb853ee171c5 in /"
-  },
-  {
-    "created": "2015-10-31T22:22:55.613815829Z",
-    "created_by": "/bin/sh -c #(nop) CMD [\"sh\"]",
-    "empty_layer": true
-  }
-]</pre>
-    </dd>
-</dl>
+- **created** *string*, OPTIONAL
+
+  A ISO-8601 formatted combined date and time at which the image was created.
+
+- **author** *string*, OPTIONAL
+
+  Gives the name and/or email address of the person or entity which created and is responsible for maintaining the image.
+
+- **architecture** *string*, REQUIRED
+
+  The CPU architecture which the binaries in this image are built to run on.
+  Possible values include: `386`, `amd64`, `arm`.
+  More values may be supported in the future and any of these may or may not be supported by a given container runtime implementation.
+  New entries SHOULD be submitted to this specification for standardization and be inspired by the [Go language documentation for $GOOS and $GOARCH](https://golang.org/doc/install/source#environment).
+
+- **os** *string*, REQUIRED
+
+  The name of the operating system which the image is built to run on.
+  Possible values include: `darwin`, `freebsd`, `linux`.
+  More values may be supported in the future and any of these may or may not be supported by a given container runtime implementation.
+  New entries SHOULD be submitted to this specification for standardization and be inspired by the [Go language documentation for $GOOS and $GOARCH](https://golang.org/doc/install/source#environment).
+
+- **config** *object*, OPTIONAL
+
+  The execution parameters which should be used as a base when running a container using the image.
+  This field can be <code>null</code>, in which case any execution parameters should be specified at creation of the container.
+
+   - **user** *string*, OPTIONAL
```

Comment:
This was previously title-case, and as of 403a1af `specs-go/v1/config.go` still has:

    User string `json:"User"`